pcdsdevices/epics_motor.py

```diff
@@ -99,14 +99,18 @@ def disable(self):
 
     def check_value(self, value):
         """
-        Check if the motor is disabled
-        Check if the value is within the soft limits of the motor.
+        Raise an exception if the motor cannot move to value.
+
+        The full list of checks done in this method:
+
+            - Check if the value is within the soft limits of the motor.
+            - Check if the motor is disabled (``.DISP`` field)
 
         Raises
         ------
-        MotorDisabledError
+        `MotorDisabledError`
```

Comment:
Missed a set of `

Reply:
Using one set instead of two creates an auto link to the definition because `default_role = any` is set in the docs conf